mongo-powerbi-connector-downloads_template.json

```diff
@@ -0,0 +1,10 @@
+{
+  "manual_link": "https://www.mongodb.com/docs/atlas/data-federation/query/query-with-sql/",
+  "release_notes_link": "https://www.mongodb.com/docs/atlas/release-notes/sql/",
+  "versions": [
+    {
+      "version": "{RELEASE_VERSION}",
```

Comment:
At some point we should standardize how we format placeholders. I've seen {}, %%, __ __, and so on.

Reply:
yes, agreed.  
Although I believe we are using {} more or less everywhere except on the reg files.
%% is on the reg file because this way we could setup our own environment variables instead of doing the search and replace and run every time. Those files were not supposed to get public, it was a developer helper which escaped (%% is not super convenient for sed command) 😅